server/users/user-sport-subscriptions.model.ts

```diff
@@ -0,0 +1,24 @@
+import { DomainModel } from "../core/domain.model"
+
+export class SportSubscriptionsModel implements DomainModel {
+	public readonly id: number | undefined
+	public readonly sportId: number
+	public readonly sportName: string | undefined
+	public readonly userId: number
+
+	constructor(
+		id: number | undefined,
+		sportId: number,
+		sportName: string | undefined,
```

Comment:
This would ideally always be defined, but if it's too expensive to fetch all sports all the time, just create a ticket and I'll refactor it later.

Reply:
Resolved in: https://github.com/mbrizic/ratethegame/pull/28/commits/87b2257ffb5cafac775bb5b87ae979372e4d59fe